functionaltest/playwright/tests/chamtechE2E.spec.ts

```diff
@@ -77,5 +77,15 @@ test.describe('three tests', () => {
 
     });
 
+    test.only(' Sample test Fossil ', async ({ page }) => {
```

Comment:
remove `.only`

Reply:
Please check i have update changes

Reply:
please check

Reply:
I have updated the spaces and test please check